docker-compose.yml

```diff
@@ -9,37 +9,41 @@ services:
     image: postgres:14-alpine
     restart: always
     volumes:
-      - db-data:/var/lib/postgresql/data
+      - ./db-data:/var/lib/postgresql/data
```

Comment:
This seems like an easy to miss breaking change. Not sure if this can be merged.

Reply:
We probably need to add versioning to this repo as well.

Reply:
Oh, indeed, seems we should add `db-data` and `event-data` to `.gitignore` file?

Reply:
If users are updating using `git pull origin master`, will this change their edited `plausible-conf.env` file? 🤔

Reply:
I don't think `plausible-conf.env` or `.gitignore` matter. Updating to this config file would use empty volume and make it appear as if all their data has been lost.

Reply:
I got it! Now I'm going to revert this part so it won't cause any breaking changes.
However in future maybe we can encourage user to use volume in this way? This is way better than using Docker `volume` when migrating.